src/LiquidationPair.sol

```diff
@@ -246,7 +246,8 @@ contract LiquidationPair is ILiquidationPair {
   function swapExactAmountOut(
     address _account,
```

Comment:
Should be renamed `_receiver` if we want to keep the same nomenclature than in the Vault.

Reply:
Yes!